cmake/ctest/drivers/atdm/ats2/drivers/Trilinos-atdm-ats2-cuda-10.1.243-gnu-7.3.1-spmpi-2019.06.24_static_opt.sh

```diff
@@ -2,4 +2,11 @@
 if [ "${Trilinos_TRACK}" == "" ] ; then
   export Trilinos_TRACK=Experimental
 fi
+export TPETRA_ASSUME_CUDA_AWARE_MPI=0
 $WORKSPACE/Trilinos/cmake/ctest/drivers/atdm/ats2/local-driver.sh
+
+export TPETRA_ASSUME_CUDA_AWARE_MPI=1
+#TODO: Trilinos-atdm-ats2-cuda-10.1.243-gnu-7.3.1-spmpi-2019.06.24_cuda-aware_static_dbg
```

Comment:
@bartlettroscoe: DO you know how to change the cdash "Build Name" here while still reusing the contents of the workspace?

Reply:
> DO you know how to change the cdash "Build Name" here while still reusing the contents of the workspace?

Yes, as described in:
 
* https://tribits.org/doc/TribitsDevelopersGuide.html#tribits-ctest-driver

I think before the second call you want to set something like:

```
export CTEST_BUILD_NAME=${JOB_NAME}-cuda-aware
export CTEST_DO_UPDATE=FALSE
export CTEST_DO_CONFIGURE=FALSE
export CTEST_DO_BUILD=FALSE
```

Reply:
I am curious, by setting these, and not doing a configure, how will the CTEST_BUILD_NAME propagate into the Ctest file? (Or is that ENV being used to override whatever Ctest finds in that file when it posts to a dashboard -- I don't know how the dashboard posting stuff works)

If this works as I think it is intended, it sounds like we aren't far from seeing this working.

Reply:
> I am curious, by setting these, and not doing a configure, how will the CTEST_BUILD_NAME propagate into the Ctest file? 

ctest -S is an independent program and has its own variables.